tests/schemas/valid/test_placeholder_in_contents_list.json

```diff
@@ -0,0 +1,78 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "0",
+  "session_timeout_in_seconds": 3,
+  "title": "Test Content Variants",
+  "theme": "default",
+  "description": "A questionnaire to test content variants and variant choices",
```

Comment:
The title and description are incorrect.

Reply:
I forgot about this🤦‍♂️